src/mjolnir/convert_transit.cc

```diff
@@ -184,39 +186,8 @@ ProcessStopPairs(GraphTileBuilder& transit_tilebuilder,
           dep.route = stop_pair.route_index();
           dep.trip = stop_pair.trip_id();
 
-          // if we have shape data then set everything, else shapeid = 0;
-          if (stop_pair.has_shape_id() && stop_pair.has_destination_dist_traveled() &&
-              stop_pair.has_origin_dist_traveled()) {
-            dep.shapeid = stop_pair.shape_id();
-            dep.orig_dist_traveled = stop_pair.origin_dist_traveled();
-            dep.dest_dist_traveled = stop_pair.destination_dist_traveled();
-          } else {
-            dep.shapeid = 0;
-          }
-
-          dep.blockid = stop_pair.has_block_id() ? stop_pair.block_id() : 0;
-          dep.dep_time = stop_pair.origin_departure_time();
-          dep.elapsed_time = stop_pair.destination_arrival_time() - dep.dep_time;
-
-          dep.frequency_end_time =
-              stop_pair.has_frequency_end_time() ? stop_pair.frequency_end_time() : 0;
-          dep.frequency =
-              stop_pair.has_frequency_headway_seconds() ? stop_pair.frequency_headway_seconds() : 0;
-
-          if (!stop_pair.bikes_allowed()) {
-            stop_no_access[dep.orig_pbf_graphid] |= kBicycleAccess;
-            stop_no_access[dep.dest_pbf_graphid] |= kBicycleAccess;
-          }
-
-          if (!stop_pair.wheelchair_accessible()) {
-            stop_no_access[dep.orig_pbf_graphid] |= kWheelchairAccess;
-            stop_no_access[dep.dest_pbf_graphid] |= kWheelchairAccess;
-          }
-
-          dep.bicycle_accessible = stop_pair.bikes_allowed();
-          dep.wheelchair_accessible = stop_pair.wheelchair_accessible();
-
-          // Compute days of week mask
+          // Compute the valid days
+          // set the bits based on the dow.// Compute days of week mask
```

Comment:
I moved the service date check right here so we abort early if there's no valid date range for a stop_pair's service